students/11th/SangwookRyu/style/main.css

```
@@ -0,0 +1,325 @@
+/* nav */
+.nav-box {
+    width: 100%;
+    height: 54px;
+    border-bottom: 1px solid #dbdbdb;
+    background-color: white;
+    position: fixed;
+    top: 0;
+}
+
+.nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    max-width: 975px;
+    height: 54px;
+    margin: 0 auto;
+    padding: 0 20px;
+    border-bottom: 1px solid #dbdbdb;
+    background-color: white;
+}
+
+.logo-box, .search-box, .btn-box {
+    display: flex;
+    max-width: auto;
+    margin: auto 0;
+    align-items: center;
+}
+
+.logo1 img {
+    width: 24px;
+    cursor:pointer;
+    vertical-align: middle;
+}
+
+.line {
+    height: 30px;
+    margin: 0 15px;
+    border-left: 2px solid black;
+}
+
+.logo2 img {
+    width: 103px;
+    position: relative;
+    top: 2px;
+    cursor: pointer;
+    vertical-align: middle;
+}
+
+.search {
+    width: 215px;
+    height: 28px;
+    padding: 7px;
+    text-align: center;
+    font-weight: 100;
+    outline: none;
+    border: 1px solid #dbdbdb;
+    border-radius: 4px;
+    background-color: #fafafa;
+}
+.search-bar::placeholder {
+    font-weight: 100;
+    color: #8e8e8e;
+}
+
+input.search:focus {
+    text-align: left;
+    color: #262626;
+}
+
+.btn-box img {
+    width: 22px;
+    margin-left: 24px;
+    cursor:pointer;
+}
+
+/* article */
+.main-box {
+    display: flex;
+    flex-direction: row;
```

```suggestion
    display: flex;
    flex-direction: row;
```
flex를 쓰는 순간 가로로 배열되지 않나요?

---

flex-direction: row 지웠습니다